### DISCUSSION TITLE
Qrexorder discussion Platform

### DISCUSSION THREAD

**Discussion by phplime** [phplime/Qrexorder#1] [Announcements]
Hi everyone,
I am starting this platform to discuss the Qqrexorder. You Can give suggestions, can discuss about the features and report the bugs, etc.
Hope this platform will help all and also me to understand your requirement.

**Comment by KewalT** [comment]
Suggestions: Can we use free push notifications services like - 1. https://github.com/novuhq/novu OR 2. https://github.com/binwiederhier/ntfy which uses Put/Post. This will help to save the cost we are paying to Pusher, One Signal.

Kindly see if you are open to accommodate this suggestion.

**Reply by phplime** [reply]
Noted with thanks. will check it.

**Comment by KewalT** [comment]
Hello phplime, 

Firstly this is amazing project & I am happy with your dedication and awesome work. 

Thank you for creating this discussion platform. I am sure it will help us to discuss efficiently & take up the things accordingly.

**Reply by phplime** [reply]
Thank you!

**Comment by nartprindi** [comment]
Hey i think i have something in mind that would make the webapp better, some features:

1.Letting the restaurant owner know which one of his staff made a order so he can know which one has the money, and if the restaurant has 2 people as staff, you can divide the tables to these people, so everytime a table has a order, the staff would who is responsibility for that table.

2.When you order from dine-in QR-Code, you don't have to check login as guest, but it does automatically.

3.You can have multi-language on the landing page, like it is in admin dashboard, restaurant dashboard and user dashboard.

4.To create a offline Point-Of-Sale(Desktop application) which is integrated with the webapp, which we can buy like the online Point-Of-Sale, because if a restaurant might have problems with WiFi they can still you that offline Point-Of-Sale.

5.Creating a IOS App and Android App, same as the web app but it is cooler to download it from the AppStore or PlayStore and there can be some features for the manager of Restaurant that can check on them even if he doesn't have WiFi.

I hope you can do these on the next update.
Thank You.

**Comment by Rpython1** [comment]
Hi sir,

I had been asking for a must-have feature for European restaurants for several months but you didn't take it into account.

Give restaurants the possibility to create mandatory and non-mandatory addons and to create categories for these addons.
For example: When the customer buys a burger, he has the option of choosing a single type of meat with a "radio button" from a list of three choices (for example, choose ONE MEAT ONLY from among: chicken, steak, turkey ).
Currently we only have NON-MANDATORY addons and we cannot create categories for these addons.

Please, take note of this feature because it is the mode of operation of all the restaurants in Europe.
I had already sent you an e-mail regarding this feature.

**Reply by phplime** [reply]
this feature is coming in the upcoming update
![SCR-20240215-tfgy](https://github.com/phplime/Qrexorder/assets/56013523/0615eea0-dea4-4274-8648-38f17f226317)

**Reply by Rpython1** [reply]
Glad to know this feature will finally be available.
The lack of this feature was a very big blocker for me.
Thank you for considering my suggestion and my needs.
Thank you so much !

**Reply by Rpython1** [reply]
When is the new update expected ?

**Reply by phplime** [reply]
Last week of March.
